compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -90,6 +90,9 @@ template <class T> class TR_Stack;
 typedef TR::Node* (* ValuePropagationPtr)(OMR::ValuePropagation *, TR::Node *);
 extern const ValuePropagationPtr constraintHandlers[];
 
+typedef TR::typed_allocator<std::pair<TR::CFGEdge *, TR_BitVector*>, TR::Region &> DefinedOnAllPathsMapAllocator;
+typedef std::map<TR::CFGEdge * const, TR_BitVector *, std::less<TR::CFGEdge *>, DefinedOnAllPathsMapAllocator> DefinedOnAllPathsMap;
```

Comment:
I think it's the `TR::typed_allocator` that needs the `TR::CFGEdge * const`, not the `std::map` :slightly_smiling_face: . See the following example: https://github.com/eclipse/omr/blob/9f86693a1d1fc70e7c998d018869d064816e8680/compiler/optimizer/CopyPropagation.hpp#L110-L113 :

Reply:
I see - changed